lighthouse-core/audits/preload-lcp.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const URL = require('../lib/url-shim.js');
+const Audit = require('./audit.js');
+const i18n = require('../lib/i18n/i18n.js');
+const MainResource = require('../computed/main-resource.js');
+const LanternLCP = require('../computed/metrics/lantern-largest-contentful-paint.js');
+const LoadSimulator = require('../computed/load-simulator.js');
+const UnusedBytes = require('./byte-efficiency/byte-efficiency-audit.js');
+
+const UIStrings = {
+  /** Title of a lighthouse audit that tells a user to preload an image in order to improve their LCP time. */
+  title: 'Preload Largest Contentful Paint image',
+  /** Description of a lighthouse audit that tells a user to preload an image in order to improve their LCP time.  */
+  description: 'Consider preloading the image used by ' +
+    'the LCP element in order to improve your LCP time.',
+};
+
+const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
+
+class PreloadLCPAudit extends Audit {
+  /**
+   * @return {LH.Audit.Meta}
+   */
+  static get meta() {
+    return {
+      id: 'preload-lcp',
```

Comment:
```suggestion
      id: 'preload-lcp-image',
```

or do we have plans for expanding this to all LCP resources?

Reply:
I don't believe we have plans beyond images